core/vibe/core/drivers/libevent2.d

```diff
@@ -1086,6 +1103,34 @@ final class Libevent2UDPConnection : UDPConnection {
 		}
 	}
 
+	void addMembership(string group)
```

Comment:
An additional commit is okay (I'm happy as long as the history stays reasonably readable).